packages/ember-views/lib/system/link-to.js

```diff
@@ -0,0 +1,7 @@
+import { symbol } from 'ember-metal/utils';
+
+// These symbols will be used to limit link-to's public API surface area.
+let HAS_BLOCK = symbol('HAS_BLOCK');
```

Comment:
These can be `export let HAS_BLOCK...` (instead of separate declaration and export statements).